independent-projects/bootstrap/maven-resolver/src/main/java/io/quarkus/bootstrap/resolver/maven/MavenArtifactResolver.java

```diff
@@ -305,7 +305,7 @@ public CollectResult collectManagedDependencies(Artifact artifact, List<Dependen
             throws BootstrapMavenException {
         try {
             return repoSystem.collectDependencies(repoSession,
-                    newCollectManagedRequest(artifact, List.of(), managedDeps, mainRepos, exclusions, Set.of(excludedScopes)));
+                    newCollectManagedRequest(artifact, directDeps, managedDeps, mainRepos, exclusions, Set.of(excludedScopes)));
```

Comment:
I had a recollection that we changed this to `List.of()` in a previous commit and I was right: https://github.com/quarkusio/quarkus/commit/bd30d5cfa498e908e4bea41c508a9da1a820b0b1#diff-99b6b872b9ffbff27f610dbe2819663a8d0b516e6f76651b4ac9fd8259b6173fR307 

Not sure if it was a mistake by then or if you actually did it on purpose?